bliss/conf/base_config.yaml

```diff
@@ -52,6 +52,7 @@ simulator:
     survey: ${surveys.sdss}
     prior: ${prior}
     n_batches: 128
+    coadd_depth: 1
```

Comment:
If `coadd_depth` is 1, is that the same as it being a raw exposure? If so, that may not be obvious to a user; a user may think that there's still some extra realignment or PSF homogenization for any coadd. So perhaps we should also have a explicit `use_coaddition` binary variable here too.

Reply:
Done.